src/doc/man/cargo-yank.md

```diff
@@ -15,9 +15,26 @@ The yank command removes a previously published crate's version from the
 server's index. This command does not delete any data, and the crate will
 still be available for download via the registry's download link.
 
-Note that existing crates locked to a yanked version will still be able to
-download the yanked version to use it. Cargo will, however, not allow any new
-crates to be locked to any yanked version.
+However, yanking a release will prevent cargo from selecting that version
+when determining the version of a dependency to use. If there are no longer
+any compatible versions that haven't been yanked, cargo will return an error.
+
+The only exception to this is crates locked to a specific version by a lockfile,
+these will still be able to download the yanked version to use it.
+
+For example, consider a crate `bar` with published versions `0.22.0`, `0.22.1`, 
+`0.22.2`, `0.23.0` and `0.24.0`. The following table identifies the versions
+cargo could use in the absence of a lockfile for different semver constraints,
+following a given release being yanked 
+
+| Yanked Version / Semver Constraint | `bar = "0.22.0"`                          | `bar = "=0.22.0"` | `bar = "0.23.0"` |
+|------------------------------------|-------------------------------------------|-------------------|------------------|
+| `0.22.0`                           | Use either `0.22.1` or `0.22.2`           | **Return Error**  | Use `0.23.0`     |
```

Comment:
Supposed Cargo chooses max SemVer-compatible version as always if no lockfile present, what "use either" means here?

Reply:
Cargo doesn't necessarily pick the most recent version, in particular if there are multiple crates with shared dependencies it tries to find a common compatible version. I've not found good documentation of this process, just witnessed it w.r.t the nix crate

Reply:
Thanks for the clarification! I misread this as a crate with only one dependency.